backends/advanced/src/advanced_omi_backend/workers/transcription_jobs.py

```
@@ -501,6 +527,293 @@ async def transcribe_full_audio_job(
+    )
+
+    logger.info(f"🔄 Enqueued batch transcription fallback job {batch_job.id}")
+
+    # Wait for batch transcription to complete
+    max_wait = timeout_seconds
+    waited = 0
+    while waited < max_wait:
+        batch_job.refresh()
+        if batch_job.is_finished:
+            if batch_job.is_failed:
+                raise Exception(f"Batch transcription failed: {batch_job.exc_info}")
```

Ruff (0.14.14)

Create your own exception

(TRY002)